voltdbclient/url.go

```diff
@@ -25,6 +25,7 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
+		// need go version >= 1.8 to use this method.
```

Comment:
The goal is to support all major go version. On my dev machine I use go1.8. I was hoping that maybe the project should have continuous integration to run the test case on all major go versions.

Probably `go>-=.4` but that needs someone to verify and I can't at the moment.